.github/workflows/lint-code.yml

```
@@ -85,3 +85,23 @@ jobs:
         with:
           isortVersion: "latest"
           requirementsFiles: "requirements.txt requirements-dev.txt"
+
+  static-type-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v3
+      with:
+        python-version: '3.11'
```

nf-core tools still supports Python 3.8 hence the type checking needs to be done for that version.

```suggestion
        python-version: '3.8'
```

---

bf60f5f2661e2baaa3bd6e0c192acd132f86a1a1